lib/widgets/message_list.dart

```diff
@@ -897,6 +897,8 @@ class MessageWithPossibleSender extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final message = item.message;
+    final messageSender =
+      PerAccountStoreWidget.of(context).users[message.senderId];
```

Comment:
nit: (two nits)
```suggestion
    final store = PerAccountStoreWidget.of(context);
    final message = item.message;
    final sender = store.users[message.senderId];
```

That way the result of `PerAccountStoreWidget.of` goes into a `store` local, like in most of our other build methods where it appears. The uniformity is mildly helpful; and this way also makes it maximally easy to add another bit of logic using the store later, without either duplicating the lookup or rewriting these lines.

Separately, most everything within this method is about the particular message `message`, so a name "sender" communicates basically the same meaning as "message sender". Similarly the existing other locals include `senderRow` and `time`, without the added qualifier "message".